### PR TITLE
change pptx files to pdf

### PR DESCRIPTION
**Pull request by markkramerus**

### CODE REVIEW

File: content/downloads/_index.md [hunk 1]
```diff
@@ -9,14 +9,16 @@ menu:
...
+* <a href="201118_Mark_Kramer_Introduction_To_FHIR_Shorthand.pdf">Introduction to FHIR Shorthand</a>
+* <a href="201118_Chris_Moesel_Lets_Build_with_FHIR_Shorthand.pdf">Let's Build with FHIR Shorthand</a>
 
 ### Tutorials from the HL7 FHIR Connectathon, September 2020
-* <a href="sept-2020-fsh-tutorial-part-1.pptx">Introduction to FHIR Shorthand</a>
-* <a href="sept-2020-fsh-tutorial-part-2.pptx">Advanced FHIR Shorthand</a>
-* <a href="sept-2020-fsh-tutorial-part-3.pptx">Wicked FHIR Shorthand (Version 2 Plans)</a>
-* <a href="sept-2020-fshing-equipment.pptx">FSHing Equipment (Tools)</a>
+
+* <a href="sept-2020-fsh-tutorial-part-1.pdf">Introduction to FHIR Shorthand</a>
```
Comment (cmoesel) [inline content/downloads/_index.md:18]: Broken link.  The PDF you uploaded is named `FHIR Shorthand Tutorial - Part 1 - September 2020.pdf `.

File: content/downloads/_index.md [hunk 2]
```diff
@@ -9,14 +9,16 @@ menu:
...
+* <a href="sept-2020-fsh-tutorial-part-2.pdf">Advanced FHIR Shorthand</a>
```
Comment (cmoesel) [inline content/downloads/_index.md:19]: Also broken link.

File: content/downloads/_index.md [hunk 3]
```diff
@@ -9,14 +9,16 @@ menu:
...
+* <a href="sept-2020-fsh-tutorial-part-3.pdf">Wicked FHIR Shorthand (Version 2 Plans)</a>
```
Comment (cmoesel) [inline content/downloads/_index.md:20]: Also broken link.

File: content/downloads/_index.md [hunk 4]
```diff
@@ -9,14 +9,16 @@ menu:
...
+* <a href="sept-2020-fshing-equipment.pdf">FSHing Equipment (Tools)</a>
```
Comment (cmoesel) [inline content/downloads/_index.md:21]: This one works! 🥳

**Review by cmoesel — CHANGES_REQUESTED** [review]
There are a few broken links due to changes in the filenames (not just extensions) when you converted them to PDF.

File: content/downloads/_index.md (hunk 3 above)
Comment (markkramerus) [reply content/downloads/_index.md:20]: Stupid stupid stupid! Fixed.

File: content/downloads/_index.md (hunk 2 above)
Comment (markkramerus) [reply content/downloads/_index.md:19]: Stupid stupid stupid! Fixed.

File: content/downloads/_index.md (hunk 1 above)
Comment (markkramerus) [reply content/downloads/_index.md:18]: Stupid stupid stupid! Fixed.

File: content/downloads/_index.md (hunk 4 above)
Comment (markkramerus) [reply content/downloads/_index.md]: Duh.

**Pull request merged**